src/main/webapp/scripts/auth.js

```diff
@@ -19,11 +19,15 @@
 const PHONE_NUMBER_LENGTH = 10;
 
 // Call to Auth servlet to get user log information
```

Comment:
Can you add JSDoc here to understand the type of `fileName`?

Reply:
Sure, thanks for the recommendation!